src/pas/plugins/ldap/plugin.py

```diff
@@ -331,7 +331,11 @@ def getGroupsForPrincipal(self, principal, request=None):
         if self.groups:
             # XXX: provide group_ids function in UGM! Way too calculation-heavy
             #      now
-            return [_.id for _ in _principal.groups]
+            try:
+                return [_.id for _ in _principal.groups]
+            except Exception as err:
+                logger.exception(err)
```

Comment:
see https://docs.python.org/3/library/logging.html#logging.Logger.exception
The ``err`` does not need to be passed, a message is expected.

Reply:
Fixed.